src/formatters/tailwind/buildTheme.js

```diff
@@ -62,6 +62,23 @@ const getColors = ({ tokens }) => {
     };
   }, {});
 };
+const getBoxShadow = ({ tokens, dictionary }) => {
+  const matchingTokens = tokens.filter(
+    (token) =>
+      token.attributes.category === "shadow" &&
+      token.attributes.type === "matrix"
+  );
+  let boxShadowObject = {};
+  matchingTokens.forEach((token) => {
+    const key = trimHyphens(token.name.replace("shadow", ""));
+    boxShadowObject[key || "DEFAULT"] = `var(--${token.name})`;
+  });
+  Object.assign(boxShadowObject, {
+    "inner-line": `inset 0 0 0 var(--${dictionary.tokens.line["width"].name}) var(--${dictionary.tokens.line["color"].name}))`,
+    "inner-line-strong": `inset 0 0 0 var(--${dictionary.tokens.line["width"].name}) var(--${dictionary.tokens.line["color-strong"].name}))`,
```

Comment:
```suggestion
    "inner-line-strong": `inset 0 0 0 var(--${dictionary.tokens.line["width"].name}) var(--${dictionary.tokens.line["color-strong"].name})`,
```

Superfluous ')' stops it from being integrated into the CSS output.